src/main/java/com/github/tomakehurst/wiremock/common/ContentTypes.java

```diff
@@ -18,12 +18,13 @@
 import static com.github.tomakehurst.wiremock.common.Strings.stringFromBytes;
 import static com.github.tomakehurst.wiremock.common.TextType.JSON;
 import static java.util.Arrays.asList;
+import static java.util.Map.entry;
+import static java.util.Map.ofEntries;
```

Comment:
A little bit unusual to do such an import, but okay I guess

Reply:
Remade without static